examples/siro_sandbox/app_states/app_state_fetch.py

```diff
@@ -125,6 +126,15 @@ def __init__(self, sandbox_service, gui_agent_ctrl, robot_agent_ctrl):
         # agent starting positions
         self._sandbox_service.env.task._fixed_starting_position = True
 
+    def _get_curr_navmesh_3d(self):
+        """Returns an array of vertex data for the triangulated NavMesh polys"""
+        largest_island_index = get_largest_island_index(
+            self.get_sim().pathfinder, self.get_sim(), allow_outdoor=False
+        )
+        return self.get_sim().pathfinder.build_navmesh_vertices(
```

Comment:
At a glance, this returns a MeshData object? I would like a list of Magnum Vector3.

Can you test this by adding an AppStateFetch `_viz_navmesh` that calls your function and uses self._sandbox_service.line_render.draw_transformed_line to draw all the lines?